### ISSUE TITLE
Compiling to Single File Executable fails due to nuget packages overwrite the same binaries.

### ISSUE THREAD

**Issue by vpenades** [emgucv/emgucv#371]
For convenience, I am using the new [Single File Executable](https://docs.microsoft.com/es-es/dotnet/core/deploying/single-file) publish feature available in the latest versions of VS.

So, when I create a project that uses Emgu.CV + Emgu.TF, I compile it, and then _I try to publish it_ , Visual studio throws this error:

    13>The "GenerateBundle" task failed unexpectedly.
    13>System.ArgumentException: Invalid input specification: Found multiple entries with the same BundleRelativePath

Which it seems it's an error that's showing up [often.](https://github.com/search?q=%22Found+multiple+entries+with+the+same+BundleRelativePath%22&type=issues)

After doing some research, it seems the GenerateBundle task requires that every file to be bundled must appear only _once_... after going through all the related issues I've found, the recomendation was to switch to VERBOSE logging when publishing, and try to find which files were being added multiple times.  This is an extract of the publish log, showing two repeated entries coming from different sources:
         
         C:\Users\xxxx\.nuget\packages\emgu.tf.runtime.windows\2.4.0.1059\build\x64\concrt140.dll
                 CopyToPublishDirectory=PreserveNewest
                 RelativePath=x64\concrt140.dll
                 TargetPath=x64\concrt140.dll
         C:\Users\xxxx\.nuget\packages\emgu.tf.runtime.windows\2.4.0.1059\build\x64\msvcp140.dll
                 CopyToPublishDirectory=PreserveNewest
                 RelativePath=x64\msvcp140.dll
                 TargetPath=x64\msvcp140.dll.
:

         C:\Users\xxxx\.nuget\packages\emgu.tf.lite.runtime.windows\2.4.0.1059\build\x64\concrt140.dll
                 CopyToPublishDirectory=PreserveNewest
                 RelativePath=x64\concrt140.dll
                 TargetPath=x64\concrt140.dll
         C:\Users\xxxx\.nuget\packages\emgu.tf.lite.runtime.windows\2.4.0.1059\build\x64\msvcp140.dll
                 CopyToPublishDirectory=PreserveNewest
                 RelativePath=x64\msvcp140.dll
                 TargetPath=x64\msvcp140.dll

As you can see, there's two repeated entries for `x64\concrt140.dll` and `x64\msvcp140.dll` coming from two different sources: `emgu.tf.runtime.windows` and `emgu.tf.lite.runtime.windows`

This is the full list of files that are giving trouble to the publish bundle task:

    x64\concrt140.dll
    x64\msvcp140.dll
    x64\msvcp140_1.dll
    x64\msvcp140_2.dll
    x64\msvcp140_codecvt_ids.dll
    x64\vccorlib140.dll
    x64\vcruntime140.dll
    x64\vcruntime140_1.dll

I understand that `emgu.cv.runtime.windows`, `emgu.tf.runtime.windows` and `emgu.tf.lite.runtime.windows` all contain the same binaries, and when used in the same project, the binaries are overwritten... something that up to now, it's been fine.

I agree that the `GenerateBundle` task should be smart enough to check that the file is exactly the same byte by byte, and only fail when the files contents are different....  but this doesn't mean better design can´t be done on Emgu's side.

So I would suggest to create a new package called Emgu.MSVC.Natives or something like that, containing these binaries, and make all other packages depend on this one. That will eliminate the binaries overwriting.

**Comment by vpenades** [comment]
Steps to reproduce:

1- Open this project:

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp3.1</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Emgu.CV.runtime.windows" Version="4.4.0.4077" />
    <PackageReference Include="Emgu.TF.runtime.windows" Version="2.4.0.1059" />
  </ItemGroup>

</Project>
```

2- Open the Publish wizard
3- Select Target Runtime: win-x64
4- Enable "Produce Single file"
5- Publish

**Comment by emgucv** [comment]
Emgu CV side has been fixed in this commit: 7c2d53768b2f32abc6cab71e46bc697a5751504c
We have created separate MSVC RT nuget packages as dependencies of the windows runtime package.
We will do the same to Emgu TF.
